CMake/cdat_modules/esmf_pkg.cmake

```diff
@@ -37,11 +37,11 @@ endif()
 if(CDAT_BUILD_ESMF_ESMP)
     if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL GNU)
       # GNU gfortran must be >= 4.3
-      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 2)
+      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 1)
```

Comment:
That is odd, I thought @chaosphere2112 had made it so that we have to have less than 5, plus 4.0 would fail with this no?